frontend/simple/views/NavBar.vue

```diff
@@ -39,45 +79,11 @@
         </div>
       </div>
     </nav>
-    <div class="modal" ref="modal" id="LoginModal">
-      <div class="modal-background" v-on:click="toggleModal"></div>
-      <div class="modal-content" style="width: 300px">
-        <div class="card is-rounded">
-          <div class="card-content">
-            <h1 class="title"><i18n>Log In</i18n></h1>
-            <div class="field">
-              <p class="control has-icon">
-                <input class="input" id="LoginName" name="name" v-model="name" v-validate @keyup.enter="login" data-vv-rules="required|regex:^\S+$" placeholder="username" required>
-                <span class="icon">
-                  <i class="fa fa-user"></i>
-                </span>
-              </p>
-              <i18n v-show="errors.has('name')" class="help is-danger">Username cannot contain spaces</i18n>
-            </div>
-            <div class="field">
-              <p class="control has-icon">
-                <input class="input" id="LoginPassword" name="password" v-model="password" v-validate @keyup.enter="login" data-vv-rules="required|min:7" placeholder="password" type="password" required>
-                <span class="icon is-small"><i class="fa fa-lock"></i></span>
-              </p>
-              <i18n v-show="errors.has('password')" class="help is-danger">Password must be at least 7 characters</i18n>
-            </div>
-            <p class="help is-danger" id="LoginResponse" v-show="response">{{response}}</p>
-            <div class="field">
-              <p class="control">
-                <button id="LoginButton" class="button is-primary is-medium is-fullwidth"
-                  @click="login" :disabled="errors.any() || !fields.passed()"
-                >
-                  <span class="icon"><i class="fa fa-user"></i></span>
-                  <i18n>Login</i18n>
-                </button>
-              </p>
-            </div>
-          </div>
-        </div>
-      </div>
-      <div class="modal-close" @click="toggleModal"></div>
-    </div>
-    <time-travel v-show="showTimeTravel" :toggleVisibility="toggleTimeTravel"></time-travel>
+    <login-modal
+      v-if="loginModalVisible"
+      @close="closeLoginModal"
+    />
```

Comment:
Personally, I prefer legibility over line length as we spend much more time reading code then writing. This format makes it easier to add/remove properties in bulk and makes it faster to parse what properties a component contains. It's somewhat a standard format in the React world. 

That said, if this isn't something you aren't open to, I can revert.